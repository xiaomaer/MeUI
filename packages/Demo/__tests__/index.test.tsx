import React from 'react';
import { render } from '@testing-library/react';
import Demo from '../demo';

describe('use demo', () => {
  test('should render default', () => {
    const { container } = render(<Demo name="xm" />);
    expect(container).toMatchSnapshot();
  });
});
