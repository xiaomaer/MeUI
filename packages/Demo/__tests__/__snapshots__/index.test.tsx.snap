// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use demo should render default 1`] = `
<div>
  <div
    class="test"
  >
    xm
  </div>
</div>
`;
